drivers/fpga/fpga_altera_agilex_bridge.c

```diff
@@ -0,0 +1,433 @@
+/*
+ * Copyright (c) 2024, Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT altr_socfpga_agilex_bridge
+
+#include <errno.h>
+#include <zephyr/device.h>
+#include <zephyr/sip_svc/sip_svc.h>
+#include <zephyr/drivers/sip_svc/sip_svc_agilex_smc.h>
+#include <zephyr/drivers/fpga.h>
+#include <zephyr/logging/log.h>
+#include "fpga_altera_agilex_bridge.h"
+
+LOG_MODULE_REGISTER(fpga_altera);
+
+static uint32_t mailbox_client_token;
+static struct sip_svc_controller *mailbox_smc_dev;
```

Comment:
These are duplicates definitions and should be removed.

Reply:
done